include/grpc/event_engine/slice_buffer.h

```diff
@@ -118,12 +125,26 @@ class SliceBuffer {
   /// associated slice.
   Slice RefSlice(size_t index);
 
+  /// Returns pointer to the buffer contained in the slice at the specified
+  /// index. The returned buffer is mutable.
+  std::tuple<uint8_t*, size_t> MutableData(size_t index) {
+    return std::make_tuple<uint8_t*, size_t>(
+        GPR_SLICE_START_PTR(slice_buffer_.slices[index]),
+        GPR_SLICE_LENGTH(slice_buffer_.slices[index]));
+  }
+
   /// The total number of bytes held by the SliceBuffer
   size_t Length() { return slice_buffer_.length; }
```

Comment:
Probably unrelated -- this could be a const member function.

Reply:
Fixed.